pkg/config/apm.go

```diff
@@ -65,6 +65,7 @@ func setupAPM(config Config) {
 	config.BindEnv("apm_config.max_events_per_second", "DD_APM_MAX_EPS", "DD_MAX_EPS")
 	config.BindEnv("apm_config.max_traces_per_second", "DD_APM_MAX_TPS", "DD_MAX_TPS")
 	config.BindEnv("apm_config.errors_per_second", "DD_APM_ERROR_TPS")
+	config.BindEnv("apm_config.disable_rare_sampler", "DD_APM_DISABLE_RARE_SAMPLER")
```

Comment:
I was thinking about adding this conf without documenting it in config_template.yaml.
This would give the ability to disable it now and leave us room in the future to either delete the RareSampler or make it's chunks per second configurable.
What do you think @gbbr ?

Reply:
Yes that's fine. There is no need to add it.